### ISSUE TITLE
The tree view for find all references is no longer accessible for NVDA users

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#99563]
CC: @isidorn 

Issue Type: <b>Bug</b>

1. press alt+shift+f12 to find all references.

Expected: NVDA correctly reads the labels, and the tree information for the selected item

Actual: part of the label appears to be truncated and not announced, no info on tree level and expanded / collapsed state announced, making this hard to use.

VS Code version: Code - Insiders 1.46.0-insider (b1ef2bf05f90dd0e6438782411ab84756f97d8f1, 2020-06-05T12:30:13.870Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (5.94GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
notes|can|1.0.1
vscode-markdownlint|Dav|0.36.0
vscode-eslint|dba|2.1.5
prettier-vscode|esb|5.0.0
rest-client|hum|0.23.2
python|ms-|2020.5.80290
remote-wsl|ms-|0.44.2
cpptools|ms-|0.28.2
Go|ms-|0.14.3
vsnotes|pat|0.7.1
rust|rus|0.7.8
markdown-preview-enhanced|shd|0.5.10
vscode-webhint|web|1.5.6

</details>

**Comment by isidorn** [comment]
Thanks for filling this and I can partialy reproduce this.
The issue is that in the references view the File elements do not have an aria label set it seems.
So I think that [here](https://github.com/microsoft/vscode/blob/04d78b2460707a53a86985d1d9375b84522a0352/src/vs/editor/contrib/gotoSymbol/peek/referencesTree.ts#L219) we return `null` for file elements.
As for the `aria-expanded` property it seems to be set properly with all the other aria elements.

@jrieken can you please look into this aria label not being returned? If you are busy let me know and I can also investigate.

**Comment by jrieken** [comment]
> @jrieken can you please look into this aria label not being returned?

We never did that for the references viewlet, the API proposal for that is pretty fresh so isn't the real question what has changes so that this regressed?

**Comment by isidorn** [comment]
@jrieken I am not when and if this was working, since I did not check this before.
@Neurrone was this working before? If so do you know when

@jrieken what API proposal do you mean? Can't we just return the name and the relative path as part for the aria label?

**Comment by jrieken** [comment]
> @jrieken what API proposal do you mean? 

Your API proposal for the aria text on tree items. The references viewlet is an extension

**Comment by isidorn** [comment]
Yeah I always forget that.
Anyways I tried with vs code stable and I see the same behavior. So it seems there is no regression here. @Neurrone can correct me if wrong
And when possible the references viewlet should jump on the new API.

**Comment by jrieken** [comment]
> And when possible the references viewlet should jump on the new API.

Sure, can do that but the contents of the reference viewlet aren't so exotic that the default shouldn't work. No codicons or such, mostly file names, paths. Maybe the highlights are trouble some here

**Comment by Neurrone** [comment]
I don't recall exactly when this broke or if it was working before. Should I go a few months back to see if this was working before?

**Comment by isidorn** [comment]
@Neurrone I checked the last stable I get the same behavior. However please note my comment that the only missing thing is for file elements, for the actual reference elements all is good for me.
If you want you can check last 2 stable releases and let us know

**Comment by Neurrone** [comment]
For the actual reference elements, they do read the label, but the level information is missing.

**Comment by alexr00** [comment]
Introduced by https://github.com/microsoft/vscode/commit/6e50f24e64dd085e373376c381253ead02069ea1

**Comment by isidorn** [comment]
@alexr00 sorry about the breakage! Are you already looking into this? I can also dive in if needed.

**Comment by alexr00** [comment]
@isidorn I'm already looking, I just dropped the commit in there so I can keep track of it!
